### ISSUE TITLE
Instructions for Linux installation

### ISSUE THREAD

**Issue by ShneekeyTheLost** [Gtaray/archipelago-monstersanctuary#1]
So, Monster Sanctuary has a native Linux release, meaning that it won't 'see' the injected winhttp.dll file. And trying to run it through Proton is... less than ideal. So, should any other Linux user wish to use this, here's the process I used to get it to function:

* Download the latest files from this github. 
* Download the latest BepInEx_unix version from BepInEx's [GitHub](https://github.com/BepInEx/BepInEx/releases/tag/v5.4.21)
* Extract files from BepInEx_unix.zip into Monster Sanctuary's folder (where the executable program is)
* IMPORTANT: Edit the extracted run_bepinex.sh script with a text editor, and where it asks for 'executible name', paste `Monster Sanctuary.x86_64` in the quotes.
* In Steam, under Launch Options, include: `./run_bepinex.sh || %command%`
* Run game from Steam. This should install BepInEx. Close game. 
* Open up the monster_sanctuary_mod.zip, then to BepInEx/plugins/ and copy the Archipelago.MonsterSanctuary folder into the BepInEx/plugins folder in your Monster Sanctuary folder. If it doesn't have a plugins folder, just copy the whole folder over. 
* Run game from Steam. Should now be properly installed and function as normal.

tl;dr: You have to use the unix version of BepInEx, make sure to open the shell script and tell it what executable to run, put in the Launch Option to run the shell script when running the game, then copy over the plugin. 

I am providing this methodology in the event any other Linux users become interested and have difficulties getting BepInEx to work, because unfortunately even if you force compatibility to run the windows version of Monster Sanctuary, the normal version of BepInEx still won't work properly, in my experience.

**Comment by kikiwako** [comment]
As a little side-note for others, do not forget to enable permission for the script 😉 :
`chmod u+x ./run_bepinex.sh`

I absolutely didn't waste at least an hour looking for fixes because of that... 😅
